bpftool/patches/0001-Making-bpftool-static.patch

```diff
@@ -0,0 +1,26 @@
+From 87ca432bc2cb7ffe8a189431153ba713b920ba8e Mon Sep 17 00:00:00 2001
+From: Manuel Buil <mbuil@suse.com>
+Date: Mon, 17 May 2021 13:55:07 +0200
+Subject: [PATCH] Making bpftool static
+
+Signed-off-by: Manuel Buil <mbuil@suse.com>
+---
+ images/bpftool/build-bpftool-native.sh | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/images/bpftool/build-bpftool-native.sh b/images/bpftool/build-bpftool-native.sh
+index 769c3c8..2deadd4 100755
+--- a/images/bpftool/build-bpftool-native.sh
++++ b/images/bpftool/build-bpftool-native.sh
+@@ -9,6 +9,8 @@ set -o pipefail
+ set -o nounset
+ 
+ cd /src/linux/tools/bpf/bpftool
++sed -i '/CFLAGS += -O2/a CFLAGS += -static' Makefile
++sed -i 's/LIBS = -lelf $(LIBBPF)/LIBS = -lelf -lz $(LIBBPF)/g' Makefile
```

Comment:
This line is not needed, `-lz` is included in bpftool LIBS variable now:

https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/tree/tools/bpf/bpftool/Makefile#n109

Reply:
good point, you told me some days ago and I forgot!